mobile/examples/whisper/android/azure/readme.md

```diff
@@ -0,0 +1,39 @@
+# Android Speech Recognition Example with Azure Custom Op
+
+This example creates a version to show how to use ORT to do speech recognition using the [Whisper](https://github.com/openai/whisper) model and calls the OpenAI Whisper endpoint using the Azure custom op. You MUST update the code to specify a valid OpenAI auth token. See [here](#prepare-an-openai-auth-token) for details.
+
+The application lets the user make an audio recording, then recognizes the speech from that recording and displays a transcript.
+
+Example App Scrennshot - A Successful result returned from OpenAI endpoint:
+
+<img width=25% src="images/Img_successful_endpoint_res.png" alt="App Screenshot" />
+
+## Set up
+
+### Prerequisites
+
+See the general prerequisites [here](../../../README.md#General-Prerequisites).
```

Comment:
the relative link needs to be updated with the new directory structure. same comment in local/readme.md

Reply:
thanks for the catch up. updated.